std_execution.bs

```diff
@@ -4555,6 +4567,71 @@ from the operation before the operation completes.
 
         - Otherwise, <code>execution::set_done(<i>GET-BASE</i>(static_cast&lt;Derived&amp;&amp;>(self)))</code>
 
+### `execution::receiver_signatures` <b>[execution.snd_rec_utils.rcvr_sigs]</b> ### {#spec-execution.snd_rec_utils.rcvr_sigs}
```

Comment:
I wonder if `completion_signatures` would be a better name than `receiver_signatures`?

Reply:
I like that.